lib/storage/src/content_manager/toc/collection_meta_ops.rs

```diff
@@ -216,8 +217,7 @@ impl TableOfContent {
                     state.key(),
                 );
             }
-
-            drop(removed);
+            removed.stop_gracefully().await;
```

Comment:
Let's still explicitly drop after


```suggestion
            removed.stop_gracefully().await;
            drop(removed);
```

Reply:
`stop_gracefully` is consuming collection